R/manual_layer_udf.R

```diff
@@ -47,13 +52,14 @@ execute_generic_udf <- function(namespace, udf=NULL, registered_udf_name=NULL, a
   if (!is.null(args)) {
     generic_udf$argument <- jsonlite::toJSON(as.integer(serialize(args, NULL)))
   }
+  if (!is.null(result_format)) {
```

Comment:
It would then save you this check as NULL is no longer a possible value.  That may or may be good (I don't know off-hand what your downstream function does).

Reply:
@eddelbuettel good call.

That actually needs to go away entirely; it was a holdover from when the default was `NULL`, but it is isn't anymore.